norddrop/src/device/mod.rs

```diff
@@ -491,6 +491,8 @@ impl NordDropFFI {
             if let Some(content_uri) = &desc.content_uri {
                 #[cfg(target_os = "windows")]
                 {
+                    let _ = content_uri;
```

Comment:
What does this achieve?

Reply:
It silences compiler warnings on Windows about unused variable `content_url`

Reply:
What do you mean by that?

Reply:
speaking about this:
```
    #[allow(unused_variables)]
```

but it's minor, lets not waste time on it

Reply:
Yeah, I've deliberately not used it, since it works on all targets. I wanted to silence the warning only on Windows and about this particular variable